src/libraries/Microsoft.NET.WebAssembly.Webcil/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+      <IsShipping>true</IsShipping>
```

Comment:
It's an [arcade SDK property](https://github.com/dotnet/arcade/blob/5730ab692bd256e6005bd7a03368cd2c219e1954/Documentation/ArcadeSdk.md?plain=1#L907).  AFAICT, You have to set `IsShipping` to true and `IsShippingPackage` to false so that the build logic will do all the normal signing, version info, etc, but skip making a nuget.